src/GridapExtensions.jl

```diff
@@ -12,7 +12,7 @@ end
 
 function instantiate_caches(x,nls::NewtonRaphsonSolver,op::NonlinearOperator)
   b = residual(op, x)
-  A = jacobian(op, x)
+  A = Gridap.jacobian(op, x)
```

Comment:
I think we should always defer to the Gridap jacobian and gradient, see comment on src/GridapTopOpt.jl

Reply:
On second thought, I think this is fine as is.